### PR TITLE
Story/external registry process

### PR DESCRIPTION
**Pull request by quirogas**
Remove the default harbor container registry
Add the default container registry option to the global config.json file

### CODE REVIEW

File: vmware_aria_operations_integration_sdk/constant.py [hunk 1]
```diff
@@ -12,10 +12,12 @@
 
 if platform == "win32":
     CONFIG_DIRECTORY = path.join(
-        environ.get("LocalAppData", ""), "VMware", "vROps Integration SDK"
+        environ.get("LocalAppData", ""), "VMware", "Aria Opperations Integration SDK"
```
Comment (kjrokos) [inline vmware_aria_operations_integration_sdk/constant.py:15]:
```suggestion
        environ.get("LocalAppData", ""), "VMware", "Aria Operations Integration SDK"
```

File: vmware_aria_operations_integration_sdk/constant.py [hunk 2]
```diff
@@ -12,10 +12,12 @@
...
     )
 else:
-    CONFIG_DIRECTORY = path.join(environ.get("HOME", ""), ".vrops-sdk")
+    CONFIG_DIRECTORY = path.join(
+        environ.get("HOME", ""), ".vmware-aria-opperations-integration-sdk"
```
Comment (kjrokos) [inline vmware_aria_operations_integration_sdk/constant.py:19]:
```suggestion
        environ.get("HOME", ""), ".vmware-aria-operations-integration-sdk"
```

File: vmware_aria_operations_integration_sdk/mp_build.py [hunk 3]
```diff
@@ -147,15 +146,22 @@ def get_docker_registry(
     **kwargs: Any,
 ) -> str:
     docker_registry = get_config_value("docker_registry", config_file=config_file)
-    default_registry_value = f"harbor-repo.vmware.com/vmware_aria_operations_integration_sdk_mps/{adapter_kind_key.lower()}"
+    default_registry_value = get_config_value("default_container_registry")
```
Comment (kjrokos) [inline vmware_aria_operations_integration_sdk/mp_build.py:149]: Do we ever set this anywhere or explain what it is? Does it make sense to have a global default?

File: vmware_aria_operations_integration_sdk/mp_build.py [hunk 4]
```diff
@@ -147,15 +146,22 @@ def get_docker_registry(
...
 
     original_value = docker_registry
     if docker_registry is None and docker_registry_arg is None:
         print(
             "mp-build needs to configure a container registry to store the adapter container image.",
             "class:information",
         )
-        docker_registry = registry_prompt(default=default_registry_value)
+        if default_registry_value is not None:
```
Comment (kjrokos) [inline vmware_aria_operations_integration_sdk/mp_build.py:157]: Do we need to verify that this value is a valid domain? If we are setting a default, we should make sure the default is valid (This is why we wanted to remove the VMware registry)

File: doc/mp-build.md [hunk 5]
```diff
@@ -48,9 +48,9 @@ If 'Other' is selected, the tool will prompt for a project path. If the path is
 
 #### Registry Credentials
 If the user is not logged into a container registry, `mp-build` will prompt
-the user to sign into one and enter their credentials. By default `mp-build` uses [harbor-repo.vmware.com](harbor-reop.vmware.com). Alternatively, the user can use 
+the user to sign into one and enter their credentials. By default `mp-build` uses the 'default_container_registry' specified in the global config.json file. Alternatively, the user can use
```
Comment (kjrokos) [inline doc/mp-build.md:51]: If we have a global default, we need to clearly explain what it is, and how it is used, and why it is useful. Also make sure we aren't confusing the config files -- 'config.json' is local, but above we're referring to a 'global' config.json file, which doesn't exist.

File: vmware_aria_operations_integration_sdk/container_versions.json [hunk 6]
```diff
@@ -2,19 +2,11 @@
...
-            "path": "powershell-adapter",
-            "version": "0.9.0"
-        }
-    ]
-}
\ No newline at end of file
+        {"language": "Java", "path": "java-adapter", "version": "0.9.0"},
+        {"language": "Powershell", "path": "powershell-adapter", "version": "0.9.0"},
+    ],
+}
```
Comment (kjrokos) [inline vmware_aria_operations_integration_sdk/container_versions.json:12]: We should try to keep changes that are unrelated to the current MR to a minimum. This looks like just formatting.

**Review by MarkArbogast — APPROVED** [review]

File: doc/mp-build.md [hunk 7]
```diff
@@ -48,7 +48,7 @@ If 'Other' is selected, the tool will prompt for a project path. If the path is
 
 #### Registry Credentials
 If the user is not logged into a container registry, `mp-build` will prompt
-the user to sign into one and enter their credentials. By default `mp-build` uses the 'default_container_registry' specified in the global config.json file. Alternatively, the user can use
+the user to sign into one and enter their credentials. By default `mp-build` uses the 'default_container_registry' specified in the global [config.json](congif.md) file. Alternatively, the user can use
```
Comment (kjrokos) [inline doc/mp-build.md:51]:
```suggestion
the user to sign into one and enter their credentials. By default `mp-build` uses the 'default_container_registry' specified in the global [config.json](config.md) file. Alternatively, the user can use
```

File: doc/config.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+```shell
+mp-build
+Building adapter [Finished]
+Waiting for adapter to start [Finished]
+mp-build needs to configure a container registry to store the adapter container image.
+Enter the tag for the container registry: projects.registry.vmware.com/vmware_aria_operations_integration_sdk/adaptername
+```
+
+
+- `projects` (array of strings): This array of strings is used by the SDK tools to track the locations of the projects. Each string value is a project location, formatted as a **UNIX** or **DOS** path (host OS). Tracking the location of each project allows the SDK tools be used anywhere in the console besides the project directory. Every time a project is created using [mp-test](mp-test.md) or built from the root directory of the project using `mp-build`, they are automatically appended to the list.
```
Comment (kjrokos) [inline doc/config.md]:
```suggestion
- `projects` (array of strings): This array of strings is used by the SDK tools to track the locations of the projects. Each string value is a project location, formatted as a **UNIX** or **DOS** path (host OS). Tracking the location of each project allows the SDK tools be used anywhere in the console besides the project directory. Any time a new project is created, tested, or built, the project is automatically appended to the list. Projects that no longer exist in the specified location are automatically removed.
```
